components/service/pocket/src/main/java/mozilla/components/service/pocket/PocketStoriesService.kt

```diff
@@ -58,4 +60,16 @@ class PocketStoriesService(
     suspend fun getStories(): List<PocketRecommendedStory> {
         return getStoriesUsecase.invoke()
     }
+
+    /**
+     * Update how many times certain stories were shown to the user.
+     *
+     * Safe to call from any background thread.
+     * Automatically synchronized with the other [PocketStoriesService] methods.
+     */
+    suspend fun updateStoriesTimesShown(updatedStories: List<PocketRecommendedStory>) {
+        if (updatedStories.isNotEmpty()) {
+            updateStoriesTimesShownUsecase.invoke(updatedStories)
+        }
+    }
```

Comment:
nit: In general this feels to be mostly a wrapper around the use case class. Maybe this could be reduced to calling the use case instead - moving the little bit of additional functionality into the use case?